pkg/cosign/keys.go

```diff
@@ -34,13 +37,19 @@ import (
 )
 
 const (
-	PrivakeKeyPemType = "ENCRYPTED COSIGN PRIVATE KEY"
-
-	BundleKey = static.BundleAnnotationKey
+	PrivateKeyPemType    = "ENCRYPTED COSIGN PRIVATE KEY"
+	RSAPrivateKeyPemType = "RSA PRIVATE KEY"
+	ECPrivateKeyPemType  = "EC PRIVATE KEY"
+	BundleKey            = static.BundleAnnotationKey
 )
 
 type PassFunc func(bool) ([]byte, error)
 
+type Key struct {
+	private crypto.PrivateKey
+	public  crypto.PublicKey
+}
+
```

Comment:
I wonder if we'd want to disambiguate key and keys here. Maybe change Keys to keybytes?

Reply:
This could be accomplished via separate PR.

Reply:
+1 when i read the function signatures below, i'm confused.

even naming the types to distinguish between crypto public/private keys and key bytes would help.